plugin/zoom/lib/API/MeetingSettings.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/* For licensing terms, see /license.txt */
+
+namespace Chamilo\PluginBundle\Zoom\API;
+
+use Exception;
+
+class MeetingSettings
```

Comment:
Codacy found an issue: [The class MeetingSettings has 27 fields. Consider redesigning MeetingSettings to keep the number of fields under 15.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5625225)